packages/foreman/dist-macro/dist-macro.spec

```diff
@@ -0,0 +1,31 @@
+%define base_release_version 7
+
+Name:     dist-macro
+Version:  1.0
+Release:  1
+
+Summary:  Package to define the dist macro
+Group:    System Environment/Base
+License:  GPLv3+
+URL:      https://theforeman.org
+
+BuildArch: noarch
+
+%description
+Sets the dist macro to basic distribution and major version.
+
+%install
+install -d -m 755 %{buildroot}/etc/rpm
+cat >> %{buildroot}/etc/rpm/dist-macro.dist << EOF
+%%rhel %{base_release_version}
```

Comment:
Where do these values come from? Is it to copy from RHEL's release?

Reply:
Look at the top of the spec file.

Reply:
I meant to comment on the line below. I don't see `%dist` and I think it's not great to depend on the build env for it.